scripts/hurl/README.md

```diff
@@ -0,0 +1,28 @@
+# Hurl Scripts
+
+## Requirements
+
+- [hurl](https://hurl.dev/)
+- [jq](https://jqlang.github.io/jq/)
+- [azure-cli](https://learn.microsoft.com/en-us/cli/azure/)
+- `CDCTI_HOME` environment variable ([see here](../README.md))
+
+## Available Hurl Scripts
+
+- [ReportStream](./rs/): scripts to send requests to ReportStream's endpoints
+- [CDC Intermediary](./ti/): scripts to send requests to the CDC Intermediary's endpoints
+- [Epic/UCSD](./epic/): scripts to send requests to Epic endpoints for UCSD
+
+## Local Submission Scripts
+
+- `submit_message.sh`: sends a HL7 message to a locally running RS instance. It also grabs the snapshots of the file in azurite after converting to FHIR, after applying transformations in TI, and after converting back to HL7. It copies these files to the same folder where the submitted file is
+    ```
+    Usage: ./submit_message.sh /path/to/message.hl7
```

Comment:
This puts `Usage:` in monospaced font too.  Did you mean that?

Reply:
Yeah, it's the output of the `help` message, but now that you mention it, it looks weird because it doesn't have anything else aside from usage. I'll remove the `Usage:` part